tests/unit/verifiers/test_result.py

```diff
@@ -72,45 +78,43 @@ def test_partial_formatting(severity):
 def test_partial_equal():
     """Test 'equal" operation between Partial instances."""
```

Comment:
unrelated to your change, but quotation around word `equal` starts with single quote and ends with double :D